modules/javafx.graphics/src/main/java/javafx/scene/text/TabStopPolicy.java

```diff
@@ -56,26 +56,25 @@ public final ObservableList<TabStop> tabStops() {
     }
 
     /**
-     * Provides default tab stops (beyond the last tab stop specified by {@code #tabStops()},
-     * as a fixed repeating distance in pixels for tabs after the last tab stop position.
-     * The position of default tab stops is computed at regular intervals relative to
-     * the leading edge of the {@code TextFlow} this policy is registered with.
+     * Specifies the default tab stop interval (beyond the last tab stop provided by {@code #tabStops()}),
+     * as a fixed repeating distance (in pixels) to the next tab stop computed at regular intervals
+     * relative to the leading edge of the {@code TextFlow} node.
      * <p>
-     * A value of less than or equal 0 disables the default stops.
+     * A value of less than or equal 0 disables the default interval.
      *
-     * @return the default tab stops property
+     * @return the default tab interval property
      * @defaultValue 0
      */
-    public final DoubleProperty defaultStopsProperty() {
-        return defaultStops;
+    public final DoubleProperty defaultInterval() {
```

Comment:
Oops. This should be `defaultIntervalProperty()`, which explains the GHA build failure.